pkg/registry/core/service/allocator/bitmap.go

```diff
@@ -70,6 +71,44 @@ func NewAllocationMap(max int, rangeSpec string) *AllocationBitmap {
 		rangeSpec: rangeSpec,
 	}
 	return &a
+
+}
+
+// reservedRangeOffset estimates the offset used on the range for statically allocation based on
+// the following formula `min(max($min, cidrSize/$step), $max)`, described as ~never less than
+// $min or more than $max, with a graduated step function between them~.
+func reservedRangeOffset(min, max, step, cidrSize int) int {
+	offset := cidrSize / step
```

Comment:
offset is zero if cidrsize < step, I have to add unit test to this function to document the corner cases